src/options/attribute-sorting/index.ts

```diff
@@ -2,7 +2,7 @@ import type { ChoiceSupportOption, PathArraySupportOption } from 'prettier';
 import { CATEGORY_PUG } from '../constants';
 
 const pugSortAttributesOption: PathArraySupportOption = {
-  since: '1.7.0',
+  // since: '1.7.0',
```

Comment:
@fisker What happend to the `since` property?
Why will it be remove in v3? Can you provide some background info?

Reply:
Does it work before? If it works, since "what"? Version of Prettier? Version of this plugin(Prettier doesn't know version of plugin)?

Even for builtin plugins. We can't find how to use it.

That's why its removed. Make sense to you?

https://github.com/prettier/prettier/pull/13699

Reply:
Previously in `@types/prettier` it was a required property (see https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d8c6e21d99e00769b450c6e091a85eb2eb4e63b3/types/prettier/index.d.ts#L554)

I always thought this was for documentation purposes :thinking: 

But if you/Prettier removed the support for it, then I can also remove it instead of just commenting them out and produce theoretically dead code